frontend/server/src/Controllers/Identity.php

```diff
@@ -433,7 +433,27 @@ public static function apiBulkCreateForTeams(\OmegaUp\Request $r): array {
                             /*$gender=*/ 'decline',
                             $teamGroup->alias
                         );
-                        \OmegaUp\DAO\Identities::create($newIdentity);
+
+                        $preexistingIdentity = \OmegaUp\DAO\Identities::findByUsername(
+                            $selfGeneratedIdentity['username']
+                        );
+                        if (is_null($preexistingIdentity)) {
+                            \OmegaUp\DAO\Identities::create($newIdentity);
+                        } else {
+                            // Check password
+                            \OmegaUp\SecurityTools::testStrongPassword(
+                                $selfGeneratedIdentity['password']
+                            );
+                            $hashedPassword = \OmegaUp\SecurityTools::hashString(
+                                $selfGeneratedIdentity['password']
+                            );
+                            $preexistingIdentity->password = $hashedPassword;
```

Comment:
```suggestion
                            $preexistingIdentity->password = $newIdentity->password;
```

esto funciona? `self::createIdentity` debería ya haber hasheado la contraseña.

Reply:
Ah, cierto. Eso funciona.